src/homepage-welcome.js

```diff
@@ -0,0 +1,18 @@
+import "./App.css";
+import NavLinks from "./components/nav-links";
+
+const Welcome = () => {
+  return (
+    <>
+      <div className="container" id="homepage">
+        <div className="wrapper">
+          <div className="home-title-1">Welcome to</div>
```

Comment:
Instead of using classes like `home-title-1` I think we could be a bit more descriptive.
In this case I would do something like `home__title` and `home__subtitle` or similarly. Can check BEM convention for CSS for more guidance https://getbem.com/naming/

Reply:
Oh right, and since these are headers, it would be better to use headings instead of a div. h1,h2,h3 etc.